### ISSUE TITLE
jQuery error on Admin View Editing within JCB after update

### ISSUE THREAD

**Issue by cpaschen** [vdm-io/Joomla-Component-Builder#280]
### Steps to reproduce the issue

Upgrade to JCB Staging on-or-around 4-26
Upgrade to JCB Beta 2.7.9
Upgrade to JCB Stable 2.7.9

Go to Admin Views in JCB
Click on an Admin View to edit

### Expected result
Admin view displays for editing

### Actual result
White screen (no content).

Console errors:
```
index.php?option=com_componentbuilder&view=admin_view&layout=edit&id=116:3 Uncaught ReferenceError: jQuery is not defined
    at index.php?option=com_componentbuilder&view=admin_view&layout=edit&id=116:3
(anonymous) @ index.php?option=com_componentbuilder&view=admin_view&layout=edit&id=116:3
jquery-noconflict.js:1 Uncaught ReferenceError: jQuery is not defined
    at jquery-noconflict.js:1
(anonymous) @ jquery-noconflict.js:1
bootstrap.js:32 Uncaught TypeError: $ is not a function
    at bootstrap.js:32
    at bootstrap.js:63
(anonymous) @ bootstrap.js:32
(anonymous) @ bootstrap.js:63
template.js:409 Uncaught ReferenceError: jQuery is not defined
    at template.js:409
(anonymous) @ template.js:409
index.php?option=com_componentbuilder&view=admin_view&layout=edit&id=116:1888 Uncaught ReferenceError: jQuery is not defined
    at index.php?option=com_componentbuilder&view=admin_view&layout=edit&id=116:1888
```

### System information (as much as possible)
- OS Name & Version:  WAMP
- MySql Version:  5.7.19
- Apache Version: 2.4.27
- PHP Version: 7.1.9
- Joomla Version: 3.8.7
- JCB Version: 2.7.9 (stable)
- Browser: Chrome

### Additional comments

This is effecting editing all admin views (including the demo LOOK admin view.
Directing editing the Admin Fields  and Admin Field Conditions (via the buttons on the Admin Views page) DOES work properly.
I have not (intentionally or knowingly) made any change to the core JCB code (other than the various updates indicated above).
The issue was happening prior to the beta version install.
Other Joomla sites running JCB, installed on the same WAMP server are NOT having the same issue.

Any ideas where I should check to figure out where this is having issues?

**Comment by cpaschen** [comment]
NOTE: I've also tried manually copying the JCB files from a working Joomla site to this one AND I went through the JCB Configuration/options to make sure they were also all the same values - no changes (still problem).

**Comment by cpaschen** [comment]
I updated the core Joomla files (i.e. reinstalled Joomla using the Joomla Update with manual update option) and that didn't fix the issue. I updated the other 2 extensions (other than JCB) that were installed on this site and still didn't fix the issue. I also re-installed JCB 2.7.9 without any improvement.

I exported the JCB Component package, created a fresh Joomla install and installed JCB 2.7.9 then imported the JCB package. Everything works as it should.

Not quite sure what caused this and why none of those steps fixed the issue, but because I can not reproduce it here anywhere, I'm going to close this as an issue not related to JCB.
